lib/fastlane/plugin/automated_test_emulator_run/actions/automated_test_emulator_run_action.rb

```diff
@@ -386,8 +386,8 @@ def self.available_options
                                        optional: false),
           FastlaneCore::ConfigItem.new(key: :SDK_path,
                                        env_name: "SDK_PATH",
-                                       description: "The path to your android sdk directory (root). ANDROID_HOME by default",
-                                       default_value: ENV['ANDROID_HOME'],
+                                       description: "The path to your android sdk directory (root). ANDROID_SDK_HOME by default",
+                                       default_value: ENV['ANDROID_SDK_HOME'] || ENV['ANDROID_HOME'],
```

Comment:
This should be `ANDROID_SDK_ROOT` instead of `ANDROID_SDK_HOME`, shouldn't it?
https://developer.android.com/studio/command-line/variables

Reply:
Ah, seems like I had a bit of a brain fade 😅